baybe/_optional/chem.py

```diff
@@ -3,9 +3,10 @@
 from baybe.exceptions import OptionalImportError
 
 try:
-    from mordred import Calculator, descriptors
-    from rdkit import Chem, RDLogger
-    from rdkit.Chem.rdMolDescriptors import GetMorganFingerprintAsBitVect
+    from rdkit import Chem
+    from skfp import fingerprints as skfp_fingerprints
```

Comment:
I think the renaming should rather happen in the importing file, not here

Reply:
why is this even renamed?

Reply:
I added it just to be clear where they are coming from, can remove renaming